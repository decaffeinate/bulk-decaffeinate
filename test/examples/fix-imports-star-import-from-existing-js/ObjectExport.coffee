module.exports = {
  firstValue: 1,
  secondValue: 2
}
