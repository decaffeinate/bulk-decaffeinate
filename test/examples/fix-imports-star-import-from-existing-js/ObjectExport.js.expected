// TODO: This file was created by bulk-decaffeinate.
// Fix any style issues and re-enable lint.
export default {
  firstValue: 1,
  secondValue: 2
};
